.test/test.sh

```diff
@@ -28,8 +28,40 @@ time bashbrew fetch "$@"
 
 time "$dir/../sources.sh" "$@" > "$dir/sources.json"
 
+rm -rf "$dir/coverage"
+mkdir -p "$dir/coverage"
+export GOCOVERDIR="${GOCOVERDIR:-"$dir/coverage"}"
+
+rm -f "$dir/../bin/builds" # make sure we build with -cover for sure
 time "$dir/../builds.sh" --cache "$dir/cache-builds.json" "$dir/sources.json" > "$dir/builds.json"
 
+# test again, but with "--cache=..." instead of "--cache ..." (which also lets us delete the cache and get slightly better coverage reports at the expense of speed / Hub requests)
+time "$dir/../builds.sh" --cache="$dir/cache-builds.json" "$dir/sources.json" > "$dir/builds.json"
```

Comment:
Why would having the equal sign or not change behavior?

Reply:
:sweat_smile: because we're currently punting on choosing a flag-parsing library:

https://github.com/docker-library/meta-scripts/blob/96ed6a9b2bad83711f6254886800675b941a4afa/builds.go#L221-L228

Also though, running it a second time _should_ give us slightly different coverage in the case of "totally fresh cache" which is useful for testing locally and has minimal impact in general thanks to our cache file).